qcodes/instrument_drivers/QuTech/S5i.py

```diff
@@ -20,34 +20,88 @@ class S5i(Instrument):
             individual modules.
 
         module (int): module number as set on the hardware.
+
+    Parameters:
+        enabled: switched output on/off
+        frequency: Frequency on Hz, range 40 MHz - 4 GHz, stepsize 1 MHz
+        freq_fine: Frequency on Hz, range 40 MHz - 4 GHz, stepsize 10 kHz.
+        Difference between frequency and freq_fine unclear!?
```

Comment:
is the difference still unclear? So far as i understand, the difference is in the step size - one is smaller than the other one. So, if the difference is clear, please, remove the line.